README.md

```diff
@@ -6,3 +6,188 @@ Distributed service to interact with development boards.
 * [boardswarm-protocol](boardswarm-protocol/README.md) - Boardswarm protocol definitions
 * [boardswarm-cli](boardswarm-cli/README.md) - Boardswarm command line client
 * [boardswarm-client](boardswarm-client/README.md) - Boardswarm client library
+
+## Installation
+
+These instructions are for configuring a simple unencrypted setup with static
+JWKS based authentication.
+
+### Building
+
+Boardswarm needs rustc version 1.70 or newer, this rules out a number of stable
+distributions for now (the kind of thing you may want to run your
+infrastructure on), however if needed we can build on a system that provides
+the required level of rust support and, as rust builds static binaries, we can
+copy them to the target server once built. As boardswarm is also quite new and
+undergoing active development, we'll build the latest from git.
+
+1. Install required tools/libraries to build boardswarm on a suitable system.
+   As we're pulling the latest from git, we'll need that installed too.
+   Assuming a Debian based system:
+   ```
+   $ sudo apt install rustc git protobuf-compiler libudev-dev
+   ```
+1. Clone the boardswarm repository:
+   ```
+   $ git clone https://github.com/boardswarm/boardswarm.git
+   ```
+1. We can now build boardswarm:
+   ```
+   $ cd boardswarm
+   $ cargo build
+   ```
+
+### Server Install
+
+The "server" is the machine that will have the development boards attached to
+it.
+
+1. Copy the `boardswarm` binary to `/usr/local/bin/` on the server.
+
+1. Create a boardswarm user and group for the boardswarm server to run as:
+   ```
+   $ sudo addgroup --system boardswarm
+   $ sudo adduser --system --ingroup boardswarm --no-create-home --disabled-password boardswarm
+   ```
+
+1. Copy the [example systemd unit](boardswarm/share/boardswarm.service) to
+   `/etc/systemd/system/boardswarm.service`.
+
+1. Copy [example udev rules](boardswarm/share/99-boardswarm.rules) to
+   `/etc/udev/rules.d/`.
+
+1. Reload udev rules and ensure they are applied:
+   ```
+   $ sudo udevadm control --reload-rules
+   $ sudo udevadm trigger
+   ```
+
+1. Copy example [`server.conf`](boardswarm/share/server.conf) to
+   `/etc/boardswarm/server.conf` on target server machine and modify as
+   required.
+
+### Server Configuration
+
+1. Install [pdudaemon](https://github.com/pdudaemon/pdudaemon) and configure it
```

Comment:
note that pdudaemon isn't mandatory; e.g. gpio actuators are available ;)  So may make sense to explain a bit more of why pdudaemon (wide support for programmable power supplies)

Reply:
Updated to mention that it's not mandatory and added more explanation